packages/react-core/src/next/components/Wizard/WizardFooter.tsx

```diff
@@ -4,60 +4,78 @@ import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/Wizard/wizard';
 
 import { Button, ButtonVariant } from '../../../components/Button';
-import { WizardControlStep, WizardNavStepFunction } from './types';
+import { isCustomWizardFooter, WizardControlStep, WizardNavStepFunction } from './types';
 
 /**
  * Hosts the standard structure of a footer with ties to the active step so that text for buttons can vary from step to step.
  */
 
 export interface WizardFooterProps {
-  /** The currently active WizardStep */
-  activeStep: WizardControlStep;
+  /** The current step */
+  currentStep: WizardControlStep;
   /** Next button callback */
   onNext: () => WizardNavStepFunction | void;
   /** Back button callback */
   onBack: () => WizardNavStepFunction | void;
   /** Cancel link callback */
   onClose: () => void;
-  /** Custom text for the Next button. The activeStep's nextButtonText takes precedence. */
+  /** Custom text for the Next button. The current step's nextButtonText takes precedence. */
   nextButtonText?: React.ReactNode;
   /** Custom text for the Back button */
   backButtonText?: React.ReactNode;
   /** Custom text for the Cancel link */
   cancelButtonText?: React.ReactNode;
-  /** Optional flag to disable the first step's back button */
+  /** Flag to disable the next button */
+  disableNextButton?: boolean;
+  /** Flag to disable the back button */
   disableBackButton?: boolean;
+  /** True to hide the Cancel button */
+  hideCancelButton?: boolean;
+  /** True to hide the Back button */
+  hideBackButton?: boolean;
 }
 
-export const WizardFooter = ({
+export const WizardFooterWrapper: React.FunctionComponent = ({ children }) => (
+  <footer className={css(styles.wizardFooter)}>{children}</footer>
```

Comment:
I am seeing a type error here.  There is no interface defined for this component.

Reply:
Are you seeing a type error locally? FunctionComponent has an interface with optional children, which is why there should be no issue here;

 ```
interface FunctionComponent<P = {}> {
        (props: PropsWithChildren<P>, context?: any): ReactElement<any, any> | null;
        propTypes?: WeakValidationMap<P> | undefined;
        contextTypes?: ValidationMap<any> | undefined;
        defaultProps?: Partial<P> | undefined;
        displayName?: string | undefined;
    }
    
type PropsWithChildren<P> = P & { children?: ReactNode | undefined };
```

Reply:
Yup.  It was locally when I pulled the  branch.